src/isDisplayable.js

```diff
@@ -0,0 +1,10 @@
+const isDisplayable = (button) => {
+  const numOrDecimal = button.type === "key";
+  const openP = button.key === "(";
+  const closedP = button.key === ")";
+  const isOperator = "/*-+".includes(button.key);
+  if (openP || closedP || numOrDecimal || isOperator) {
+    return true;
+  }
```

Comment:
Should we return false?

Reply:
@grimxreaper Definitely! It's super important to try and return the same data type from a function in all cases so that you don't have to go into the function to know what it will return. Granted, undefined would evaluate to false. But returning false shows your intention.

Reply:
Really good to know, great!